app/src/features/FhirResourceTree/resourceTreeUtils.ts

```diff
@@ -261,3 +261,30 @@ export const getNode = (
   }
   return undefined;
 };
+
+export const findChildAttributes = (
+  attributeToDelete: Attribute,
+  attributes: Attribute[]
+): Attribute[] => {
+  const path = attributeToDelete.path;
+  return attributes.filter((attribute) => attribute.path.startsWith(path));
+};
+
+/**
+ * Returns the first available index from the parent node path
+ * ie: if parent has 3 children and their path indexes are respectively 1, 2 & 3
+ * it will return 0.
+ * If all path indexes are taken by the parent children, returns parent.childen.length
```

Comment:
if we have a node with 2 or more children and we delete the first for example, would it be worth it to decrement the index of the next children ? here we would just return `parent.childen.length`

Reply:
I don't think it would be worth it indeed. I discussed about it with @simonvadee, and the same choice have been made in the previous Pyrog version.

The real problem would not be about updating the siblings' paths, but for those siblings' children attributes...
ie : If we change `Identifier[1]` into `Identifier[0]`, we'll have to check for any attributes which path starts with `Identifier[1]`, like `Identifier[1].type.coding[0]` and then update it into `Identifier[0].type.coding[0]`

Reply:
Unicity of paths is more important than index logic in out case